### DISCUSSION TITLE
GDL website is back

### DISCUSSION THREAD

**Discussion by slayoo** [gnudatalanguage/gdl#875] [General]
Thanks to @pjb7687, we have the GDL website running at https://gnudatalanguage.github.io/
@gnudatalanguage/gdlteam feedback welcome!

**Comment by slayoo** [comment]
One can easily contribute by making PRs to https://github.com/gnudatalanguage/gnudatalanguage.github.io
Merging the PRs will automatically deploy the new material making it visible at gnudatalanguage.github.io

**Comment by GillesDuvert** [comment]
Awesome! Thanks @pjb7687
